src/Xamarin.Android.Build.Tasks/Utilities/NativeAssemblyGenerator/NativeAssemblerAttribute.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+
+namespace Xamarin.Android.Tasks
+{
+	[AttributeUsage (AttributeTargets.Field | AttributeTargets.Property, Inherited = true)]
+	class NativeAssemblerAttribute : Attribute
+	{
+		public bool Ignore			 { get; set; }
+		public bool UsesDataProvider { get; set; }
+		public string? Comment		 { get; set; }
+		public string? Name			 { get; set; }
```

Comment:
You should use spaces after non-whitespace characters to ensure appropriate vertical alignment.